### PR TITLE
Added Kafka Connect JDBC source Mysql

### LINKED ISSUE #422: Migrating Kafka Connect JDBC Mysql Article
# What should we add to our content collections?

Migrating Kafka Connect JDBC Mysql Article
https://help.aiven.io/en/articles/3427922-jdbc-source-connector-with-mysql

# Is this something you would work on yourself?

[X] I plan to open a pull request for this

### PR DESCRIPTION
**Pull request by ftisiot**
# What changed, and why it matters

Fixes #422

### CODE REVIEW

File: docs/products/kafka/kafka-connect/howto/jdbc-source-connector-mysql.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+To setup a JDBC source connector pointing to MySQL, you need an Aiven for Apache Kafka service :doc:`with Kafka Connect enabled <enable-connect>` or a :ref:`dedicated Aiven for Apache Kafka Connect cluster <apache_kafka_connect_dedicated_cluster>`. 
+
+Furthermore you need to collect the following information about the source MySQL database upfront:
+
+* ``MYSQL_HOST``: The database hostname
+* ``MYSQL_PORT``: The database port
+* ``MYSQL_USER``: The database user to connect
+* ``MYSQL_PASSWORD``: The database password for the ``MYSQL_USER``
+* ``MYSQL_DATABASE_NAME``: the database name
```
Comment (laysauchoa) [inline docs/products/kafka/kafka-connect/howto/jdbc-source-connector-mysql.rst:23]: I think is good to be consistent. you either start all the sentences with Caps or not.

Comment (ftisiot) [reply docs/products/kafka/kafka-connect/howto/jdbc-source-connector-mysql.rst:23]: Fixed, thanks!

File: docs/products/kafka/kafka-connect/howto/jdbc-source-connector-mysql.rst [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+* ``topic.prefix``: the prefix that will be used for topic names. The resulting topic name will be the concatenation of the ``topic.prefix`` and the schema and table name.
+* ``tasks.max``: maximum number of tasks to execute in parallel. By default is 1, the connector can use at max 1 task for each source table defined.
+* ``poll.interval.ms``: query frequency, default 5000 milliseconds
+
+Check out the `dedicated documentation <https://github.com/aiven/jdbc-connector-for-apache-kafka/blob/master/docs/source-connector-config-options.rst>`_ for the full list of parameters.
+
+Create a Kafka Connect connector with Aiven CLI
+'''''''''''''''''''''''''''''''''''''''''''''''
+
+To create the connector, execute the following :ref:`Aiven CLI command <avn_service_connector_create>`, replacing the ``SERVICE_NAME`` with the name of the Aiven service where the connector needs to run:
```
Comment (laysauchoa) [inline docs/products/kafka/kafka-connect/howto/jdbc-source-connector-mysql.rst:69]: where the connector needs to run:

is not the user who has to run this?

Comment (ftisiot) [reply docs/products/kafka/kafka-connect/howto/jdbc-source-connector-mysql.rst:69]: The connector runs in Aiven, in the Kafka cluster or in a dedicated Kafka connect cluster

File: docs/products/kafka/kafka-connect/howto/jdbc-source-connector-mysql.rst [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+.. Tip::
+
+    Sourcing data from a database into Apache Kafka decouples the database from the set of consumers. Once the data is in Apache Kafka, multiple applications can access it without adding any additional query overhead to the source database.
+
+.. _connect_jdbc_mysql_source_prereq:
+
+Prerequisites
+-------------
+
+To setup a JDBC source connector pointing to MySQL, you need an Aiven for Apache Kafka service :doc:`with Kafka Connect enabled <enable-connect>` or a :ref:`dedicated Aiven for Apache Kafka Connect cluster <apache_kafka_connect_dedicated_cluster>`.
```
Comment (anelook) [inline docs/products/kafka/kafka-connect/howto/jdbc-source-connector-mysql.rst:15]:
```suggestion
To setup a JDBC source connector pointing to MySQL, you need an Aiven for Apache Kafka service :doc:`with Apache Kafka Connect enabled <enable-connect>` or a :ref:`dedicated Aiven for Apache Kafka Connect cluster <apache_kafka_connect_dedicated_cluster>`. 
```

File: docs/products/kafka/kafka-connect/howto/jdbc-source-connector-mysql.rst [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+Example: define a JDBC incremental connector
+--------------------------------------------
+
+The example creates an :doc:`incremental <../concepts/jdbc-source-modes>` JDBC connector with the following properties:
+
+* connector name: ``jdbc_source_mysql_increment``
+* source tables: ``students`` and ``exams``, available in an Aiven for MySQL database 
+* :doc:`incremental column name <../concepts/jdbc-source-modes>`: ``id``
+* topic prefix: ``jdbc_source_mysql_increment.``
+* maximum number of concurrent tasks: ``1``
```
Comment (anelook) [inline docs/products/kafka/kafka-connect/howto/jdbc-source-connector-mysql.rst:96]:
```suggestion
* maximum number of concurrent tasks: 1
```

**Review by anelook — COMMENTED** [review]
Looks good to me, only a couple of minor changes, otherwise it is very similar to PG. 
But I like that we keep it separately.

**Review by anelook — APPROVED** [review]

**Pull request merged**